src/rp2/abstract_accounting_method.py

```diff
@@ -69,11 +93,15 @@ def set_partial_amount(self, acquired_lot: InTransaction, amount: RP2Decimal) ->
     def clear_partial_amount(self, acquired_lot: InTransaction) -> None:
         self.set_partial_amount(acquired_lot, ZERO)
 
-    def __iter__(self) -> "AccountingMethodIterator":
-        return AccountingMethodIterator(self.__acquired_lot_list, self.__from_index, self.__to_index, self.__accounting_method.lot_candidates_order())
+    def add_selected_lot_to_heap(self, lot: InTransaction) -> None:
```

Comment:
This looks like a protected rathen than public method: it should start with `_`.